include/core/lethegridtools.h

```diff
@@ -213,6 +213,12 @@ namespace LetheGridTools
       &                    object,
     const Point<spacedim> &trial_point);
 
+  template <int dim>
+  std::vector<typename DoFHandler<dim>::active_cell_iterator>
+  find_boundary_cell_in_sphere(const DoFHandler<dim> &dof_handler,
+                               const Point<dim> &     center,
+                               double                 radius);
+
```

Comment:
Can you add Brief + the explanation of the parameter to explain what is the goal of this function.